Library/Homebrew/os/mac.rb

```diff
@@ -214,6 +214,7 @@ def preferred_arch
       "6.3"   => { :clang => "6.1", :clang_build => 602 },
       "6.3.1" => { :clang => "6.1", :clang_build => 602 },
       "6.3.2" => { :clang => "6.1", :clang_build => 602 },
+      "7.0"   => { :clang => "6.1", :clang_build => 602 },
```

Comment:
Unless you mean the `clang_build` should be `700`? Apologies for the confusion :smile_cat: